azure-pipelines.yml

```diff
@@ -24,62 +24,3 @@ stages:
   - template: pipeline-templates/job/code-build.yml
   - template: pipeline-templates/job/unittests.yml
 
-- stage: Deploy_DST
```

Comment:
Are these deploy stages now being handled by a separate Azure DevOps pipeline?

Reply:
Advised by Praveen that a new pipeline has been created:
https://dev.azure.com/sfa-gov-uk/DCT/_release?_a=releases&view=mine&definitionId=623